src/cli/cli.go

```diff
@@ -50,6 +51,13 @@ func preRun(cc *cobra.Command, args []string) error {
 		flagSl = append(flagSl, f)
 	}
 
+	avoidTheseCommands := []string{
+		"corso", "env", "help", "backup", "details", "list", "restore", "delete", "repo", "init", "connect",
+	}
+	if len(logger.LogFile) > 0 && !slices.Contains(avoidTheseCommands, cc.Use) {
```

Comment:
Maybe we should have the check `len(logger.LogFile) == 0` check to `PreloadLoggingFlags` and have it use the default.

Reply:
The default should already be established by this point, iirc.